test/testenv/verificationutils.go

```diff
@@ -489,3 +502,38 @@ func VerifySplunkServerConfSecrets(deployment *Deployment, testenvInstance *Test
 		}
 	}
 }
+
+// VerifyPVCs verifies if PVC exist
+func VerifyPVCs(deployment *Deployment, ns string, PvcName string, expectedPVC string, expectedtoExist bool) {
+	gomega.Eventually(func() bool {
+		result := false
+		pvcexists := false
+		output, err := exec.Command("kubectl", "get", "pvc", "-n", ns, "-o", "json").Output()
+		if err != nil {
+			cmd := fmt.Sprintf("kubectl get pvc -n %s -o json", ns)
+			logf.Log.Error(err, "Failed to execute command", "command", cmd)
+			return false
+		}
+		restResponse := PvcDetailsStruct{}
+		err = json.Unmarshal([]byte(output), &restResponse)
+		if err != nil {
+			logf.Log.Error(err, "Failed to parse JSON")
+			return false
+		}
+
+		for i := 0; i < len(restResponse.Items); i++ {
+			if strings.Contains(restResponse.Items[i].Metadata.Name, expectedPVC) {
+				pvcexists = true
+				break
+			}
+		}
+
+		if expectedtoExist && pvcexists || !expectedtoExist && !pvcexists {
+			result = true
+			logf.Log.Info("PVC status (exists or not) is as expected: ", "PVC", PvcName, "STATUS", pvcexists, "EXPECTED", expectedtoExist)
+		} else {
+			logf.Log.Info("Incorrect PVC status (Exists when it shouldn't or the opposite): ", "PVC", PvcName, "STATUS", pvcexists, "EXPECTED", expectedtoExist)
+		}
+		return result
+	}, VerificationTimeout, PollInterval).Should(gomega.Equal(true))
```

Comment:
Instead of `VerificationTimeout` you can accept an additional parameter which can be customised based on test scenario.